core/src/main/java/tech/tablesaw/api/LongColumn.java

```diff
@@ -409,9 +409,27 @@ public long get(int index) {
         return data.getLong(index);
     }
 
+    @Override
+    public int getInt(int index) {
+        double value = data.getLong(index);
+        return value == LongColumn.MISSING_VALUE ? IntColumn.MISSING_VALUE : (int) value;
```

Comment:
As noted by the build bot, it's not necessary to fully qualify MISSING_VALUE here. Similar issues elsewhere